cdap-master/pom.xml

```diff
@@ -388,20 +388,20 @@
                 </configuration>
               </execution>
               <execution>
-                <id>copy-rg</id>
+                <id>copy-program-report-app-spark2_2.11</id>
                 <phase>process-resources</phase>
                 <goals>
                   <goal>copy-resources</goal>
                 </goals>
                 <configuration combine.self="override">
-                  <outputDirectory>${stage.artifacts.dir}</outputDirectory>
+                  <outputDirectory>${stage.artifacts.dir}/spark2_2.11</outputDirectory>
```

Comment:
we should use spark version variable instead of hardcoding everwhere.

Reply:
we cannot hardcode this, since we need to build jars for both Spark version under both spark1 and spark2 folders, like data pipeline

Reply:
yeah, i am just saying put this under spark2 pom file variable, why isn't that possible ?

Reply:
oh, I misunderstood what you meant